### DISCUSSION TITLE
Why we call it fast_SD and what's the difference from Automatic1111

### DISCUSSION THREAD

**Discussion by sanghyun0927** [TheLastBen/fast-stable-diffusion#586] [Q&A]
Sorry to be dumb, Absolute newbie here.

Can some answer the reason we call it fast-SD and What is the difference between this repository and automatic 1111

**Comment by TheLastBen** [comment]
A1111 is a webui repo and this is a colab that uses A1111.
I called it fast back in the day when xformers was fresh out and this colab was the only one that implemented it.
